resources/css/canvas.scss

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2015 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+.canvas-container {
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+
+    &::-webkit-scrollbar,
+    &::-webkit-scrollbar-corner {
+        width: 20px;
+        height: 5px;
+    }
+
+    &::-webkit-scrollbar-thumb {
+        background: #607d8b;
+        border-radius: 14px;
+        border: 5px solid white;
+    }
+    &::-webkit-scrollbar-thumb:hover {
+        background: #455a64;
+    }
+
+    canvas {
+        background-color: white;
+        border: 1px solid black;
```

Comment:
I'll try to move all "border"-logic to the same place in the drawing function so it's not defined at different places.